DecipherNow/gm-fabric-dashboard
# 408 group and sorting

This PR implements Groups, Views, and Sorting controls in the Fabric Table Toolbar.

To test,
1. Run `npm mock-sds`
2. In `public/index.html`, set the fabricServer meta tag equal to the following:
`<meta property="fabricServer" content="http://localhost:1337">`
3. In a separate tab, run `npm start`. This should render the following:
![image](https://user-images.githubusercontent.com/8700322/30933275-193ac198-a398-11e7-955e-123e927ceb25.png)
4. In the Search Input, search 'defense' and it should filter out only the second batcave microservice. Clear the search bar to see all services.
5. In the Cards / Table toggle, make sure that the selected toggle is gray when you click on it. It's a controlled component
6. In the Group and Sort dropdowns, select the various options in the dropdown and make sure that the secondary string updates to the last things selected.

Finally, review my code. :)
Don't be spooked by the number of file changes, as I updated the icon SVG files, and they do not need to be reviewed.

Thanks!!!
